src/test/java/com/capstone/producer/TfrHandlerTest.java

```diff
@@ -0,0 +1,141 @@
+package com.capstone.producer;
+
+import com.capstone.producer.kafka.KafkaProducer;
+import com.fasterxml.jackson.core.JsonProcessingException;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.MockedStatic;
+import org.mockito.junit.MockitoJUnitRunner;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.Mockito.*;
+
+@RunWith(MockitoJUnitRunner.class)
+public class TfrHandlerTest {
+
+    @InjectMocks
+    private TfrHandler tfrHandler;
+
+    private String boundaryPart1 = "FDC 4/3473 ZAB PART 1 OF 2 TX..AIRSPACE VAN HORN, TX..TEMPORARY\r\n" + //
```

Comment:
I think Java 17 supports text blocks which might be more convenient for this type of multiline strings.